manifests/install.pp

```diff
@@ -2,21 +2,24 @@
 #
 # @api private
 class qpid::install {
-  include qpid::tools
```

Comment:
I dropped this (and the before later on) as this should only be needed when running a qpid-config command which is handled by the config_cmd. If this is kept in, idempotency is broken when `ensure => absent` is set. We would then need to define the class here, and pass in a dedicated ensure to the `qpid::tools` class additionally. This does mean the qpid-tools package won't be removed but given that is a client tool I'm not too worried about it when wanting to ensure qpid the service is absent.

If a reviewer feels we should be also removing that package I'll add a parameter to the `qpid::tools` class and pass it in here.

Reply:
The work involved with removing it seems small enough that I am in favor of leaving no package behind when we clean up.